WMF Framework/Event Platform Client/EventPlatformClientLibrary.swift

```diff
@@ -329,17 +328,18 @@ public class EPC {
          * Refer to mw:Wikimedia_Product/Analytics_Infrastructure/Stream_configuration#Stream_cc-ing
          * for additional documentation.
          */
+        var copies = [String: [String]]()
         for stream in config.keys {
             let s = stream.split(separator: ".")
             let nPrefixes = s.count - 1
             if nPrefixes > 1 {
                 for i in 1...nPrefixes {
                     let child = s[0...i].joined(separator: ".")
                     let parent = s[0..<i].joined(separator: ".")
-                    streamCopy.appendIfNew(key: parent, value: child)
+                    copies.appendIfNew(key: parent, value: child)
                 }
             } else if nPrefixes == 1 {
-                streamCopy.appendIfNew(key: String(s[0]), value: stream)
+                copies.appendIfNew(key: String(s[0]), value: stream)
             }
         }
 
```

Comment:
You probably want `let jsonString = try copies.toPrettyPrintJSONString()` on line 348 below now.